Dism/Dism.cpp

```diff
@@ -83,21 +85,27 @@ extern "C" UINT __stdcall Dism(MSIHANDLE hInstall)
 	// Parse CAD
 	while ((hr = WcaReadStringFromCaData(&szCAD, &currState.szInclude)) != E_NOMOREITEMS)
 	{
-		ExitOnFailure(hr, "Failed getting CustomActionData field");
+		ExitOnFailure(hr, "Failed getting EnableFeatures CustomActionData field");
 
 		hr = WcaReadStringFromCaData(&szCAD, &currState.szExclude);
-		ExitOnFailure(hr, "Failed getting CustomActionData field");
+		ExitOnFailure(hr, "Failed getting ExcludeFeatures CustomActionData field");
 
 		hr = WcaReadStringFromCaData(&szCAD, &currState.szPackage);
-		ExitOnFailure(hr, "Failed getting CustomActionData field");
+		ExitOnFailure(hr, "Failed getting PackagePath CustomActionData field");
 
 		hr = WcaReadIntegerFromCaData(&szCAD, &currState.nMsiCost);
-		ExitOnFailure(hr, "Failed getting CustomActionData field");
+		ExitOnFailure(hr, "Failed getting Cost CustomActionData field");
 
 		hr = WcaReadIntegerFromCaData(&szCAD, (int*)&currState.eErrorHandling);
-		ExitOnFailure(hr, "Failed getting CustomActionData field");
+		ExitOnFailure(hr, "Failed getting ErrorHandling CustomActionData field");
+
+		hr = WcaReadIntegerFromCaData(&szCAD, (int*)&currState.bEnableAll);
+		ExitOnFailure(hr, "Failed getting EnableAll CustomActionData field");
 
-		hr = MemInsertIntoArray((void**)&pStates, 0, 1, dwStateNum, sizeof(ProgressReportState), 1);
+		hr = WcaReadIntegerFromCaData(&szCAD, (int*)&currState.nOrder);
```

Comment:
Is there any use for the Order field here?
Usually I only use it in the ORDER BY clause of the SQL statement